Pipfile

```diff
@@ -16,8 +16,9 @@ requests = "*"
 scipy = "*"
 matplotlib = "*"
 ruptures = "*"
-bias-correction = "*"
 jupyter = "*"
+bias-correction = {git = "https://github.com/pankajkarman/bias_correction.git"}
+xarray = "*"
```

Comment:
Is `Xarray` used anywhere in the code?  I couldn't spot it if so!

Reply:
It is used (and required) by the bias-correction package but for some reason it's not listed as a dependency in the GitHub version of that package. I had to use that GitHub version because the PyPi version wasn't handling the quantile mapping how it should have been, causing an issue for places with > 50% calm observations.